src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/Repeater/RepeaterTests.cs

```diff
@@ -201,7 +230,7 @@ public void ValidateGetSetBackground()
 		}
 
 		[TestMethod]
-		[Ignore("UNO: Test does not pass yet with Uno https://github.com/unoplatform/uno/issues/4529")]
+		[Ignore("Fails")]
```

Comment:
Still https://github.com/unoplatform/uno/issues/4529 ?

Reply:
Yes, but I am keeping it in mind, I will be pushing another PR with fixes for the tests, just wanted to have this leak merged for release.